ValheimPlus/GameClasses/ObjectDB.cs

```diff
@@ -0,0 +1,249 @@
+            // { "$se_mead_name", "tastyMead" }, 
+            // { "$item_mead_hp_medium", "mediumHealingMead" },
+            // { "$item_mead_hp_minor", "minorHealingMead" },
+            // { "$item_mead_stamina_medium", "mediumStaminaMead" },
+            // { "$item_mead_stamina_minor", "minorStaminaMead" },
+        };
+
+        [HarmonyPatch(typeof(ObjectDB), "Awake")]
+        public static class ModifyEffects
+        {
+            private static void Postfix(ObjectDB __instance)
+            {
+                if (Configuration.Current.Effects.IsEnabled)
+                {
+                    ConfigureEffects(__instance);
+                }
+            }
+        }
+        private static Skills.SkillType ParseAttackSkill(string attackSkill)
+        {
+            try
+            {
+                return (Skills.SkillType)Enum.Parse(typeof(Skills.SkillType), attackSkill.Trim(), true);
+            }
+            catch (ArgumentException)
+            {
+                return Skills.SkillType.None;
+            }
+        }
+
+        private static HitData.DamageType ParseDamageType(string damageType)
+        {
+            try
+            {
+                return (HitData.DamageType)Enum.Parse(typeof(HitData.DamageType), damageType.Trim(), true);
+            }
+            catch (ArgumentException)
+            {
+                return HitData.DamageType.Physical;
+            }
+        }
+
+        private static HitData.DamageModifier ParseDamageModifier(string damageModifier)
+        {
+            try
+            {
+                return (HitData.DamageModifier)Enum.Parse(typeof(HitData.DamageModifier), damageModifier.Trim(), true);
+            }
+            catch (ArgumentException)
+            {
+                return HitData.DamageModifier.Ignore;
+            }
+        }
+
+        private static List<HitData.DamageModPair> ParseDamageTypesModifiers(string damageTypesModifiers)
+        {
+            string[] damageTypeModifiers;
+            List<HitData.DamageModPair> damageTypesModifiersListing = new List<HitData.DamageModPair>();
+            damageTypeModifiers = damageTypesModifiers.Split('|');
+            foreach (string damageTypeModifier in damageTypeModifiers)
+            {
+                string[] damageTypeModifierConfiguration = damageTypeModifier.Split(':');
+                if (damageTypeModifierConfiguration.Length == 2)
+                {
+                    HitData.DamageModPair damageModPair;
+                    damageModPair.m_type = ParseDamageType(damageTypeModifierConfiguration[0]);
+                    damageModPair.m_modifier = ParseDamageModifier(damageTypeModifierConfiguration[1]);
+                    damageTypesModifiersListing.Add(damageModPair);
+                }
+            }
+            return damageTypesModifiersListing;
+        }
+
+        private static void DisplayEffectStats(SE_Stats currentStatus, bool isBefore)
```

Comment:
Can we have less debug prints ? 

The debug option makes sense in dev, not in release